README.md

```diff
@@ -1,68 +1,333 @@
 
-      npm install sendbird --save
+### More about Sendbird Chat SDK for JavaScript
 
-[Download](https://github.com/sendbird/Sendbird-SDK-JavaScript) for [Sendbird](https://sendbird.com) JavaScript SDK if you wish to use it without a package manager.
+Find out more about Sendbird Chat for JavaScript on [Chat SDK for JavaScript doc](https://sendbird.com/docs/chat/v3/javascript/getting-started/about-chat-sdk). 
 
-> NOTICE! You should also add [axios](https://github.com/axios/axios) library into the `script` tag before the Sendbird library inclusion since `v3.0.127`. You can host the library by your own, or utilize the CDN as the library refers to it. The below is an example.
+<br />
 
-```html
-<script src="https://unpkg.com/axios/dist/axios.min.js"></script>
-<script src="/your-own-path-to-sendbird/Sendbird.min.js"></script>
+## Before getting started
+
+This section shows you the prerequisites you need to check for using Sendbird Chat SDK for JavaScript.
+
+### Supported browsers
+
+| Browser | Supported versions |
+| :---: | :--- |
+| Internet Explorer | 10 or higher |
+| Edge | 13 or higher |
+| Chrome | 16 or higher |
+| Firefox | 11 or higher |
+| Safari | 7 or higher |
+| Opera | 12.1 or higher |
+| iOS Safari | 7 or higher |
+| Android Browswer | 4.4 (Kitkat) or higher |
+
+### More about additional features of Sendbird the the Chat SDK
+
+Try building your Sendbird application with these two add-ons:
+
+- [Sendbird UIKit for JavaScript](https://sendbird.com/docs/uikit/v1/javascript/getting-started/about-uikit): a development kit with a user interface that enables an easy and fast integration of standard chat features into new or existing client apps.
+- [Sendbird SyncManager for JavaScript](https://sendbird.com/docs/syncmanager/v1/javascript/getting-started/about-syncmanager): the Chat SDK add-on that optimizes the user caching experience by interlinking the synchronization of the local data storage with the chat data in Sendbird server through an event-driven structure. 
+
+<br />
+
+## Getting started
+
+This section gives you information you need to get started with Sendbird Chat SDK for JavaScript.
+
+### Try the sample app
+
+The fastest way to test the Chat SDK is to build your chat app on top of our sample app. To create a project for the sample app, download the app from our GitHub repository. The link is down below. 
+
+- https://github.com/sendbird/Sendbird-JavaScript
+
+You can also download the sample using a git command:
+
+```bash
+$ git clone https://github.com/sendbird/SendBird-JavaScript
```

Comment:
It'd be good to add the live sample page https://sample.sendbird.com imho.

Reply:
Done!